tests/fixture-valid.js

```diff
@@ -398,6 +389,111 @@ module.exports = function checkFixture(manKey, fixKey, fixtureJson, uniqueValues
 
         return values;
       }
+
+      /**
+       * Check that a capability is valid (except its dmx range).
+       * @param {!Capability} cap The capability to check.
+       * @param {!string} errorPrefix An identifier for the capability to use in errors and warnings.
+       */
+      function checkCapability(cap, errorPrefix) {
+        const switchingChannelAliases = Object.keys(cap.switchChannels);
+        if (!arraysEqual(switchingChannelAliases, channel.switchingChannelAliases)) {
+          result.errors.push(`${errorPrefix} must define the same switching channel aliases as all other capabilities.`);
+        }
+        else {
+          switchingChannelAliases.forEach(alias => {
+            const chKey = cap.switchChannels[alias];
+            usedChannelKeys.add(chKey.toLowerCase());
+
+            if (channel.fixture.getChannelByKey(chKey) === null) {
+              result.errors.push(`${errorPrefix} references unknown channel '${chKey}'.`);
+            }
+          });
+        }
+
+        const startEndEntities = Capability.START_END_PROPERTIES.map(
+          prop => [prop, cap[prop]]
+        ).filter(
+          ([prop, value]) => value !== null
+        );
+
+        for (const [prop, [startValue, endValue]] of startEndEntities) {
+          if ((startValue.keyword === null) !== (endValue.keyword === null)) {
+            result.errors.push(`${errorPrefix} must use keywords for start and end value or for none of them in ${prop}.`);
+          }
+          else if (startValue.unit !== endValue.unit) {
+            result.errors.push(`${errorPrefix} uses different units for ${prop}.`);
+          }
+        }
+
+        const capabilityTypeChecks = {
+          ShutterStrobe: () => {
+            const hasSpeed = cap.speed !== null;
+            const hasDuration = cap.duration !== null;
+
+            if ([`Closed`, `Open`].includes(cap.shutterEffect) && (hasSpeed || hasDuration)) {
+              result.errors.push(`${errorPrefix}: Shutter open/closed can't define speed or duration.`);
+            }
+
+            if (hasSpeed && hasDuration) {
+              result.errors.push(`${errorPrefix} can't define both speed and duration.`);
+            }
+          },
+          Pan: checkPanTiltCapability,
+          Tilt: checkPanTiltCapability,
+          PanContinuous: () => checkPanTiltContinuousCapability(`Pan`),
```

Comment:
You can just pass `checkPanTiltContinuousCapability` like the other functions, and check `cap.type` instead of the parameter there.